server/app/src/main/resources/db/migration/V1__CreateTablesSchema.sql

```diff
@@ -0,0 +1,76 @@
+
+CREATE TABLE IF NOT EXISTS `Student` (
+  `id` int(10) NOT NULL AUTO_INCREMENT,
+  `username` varchar(20) NOT NULL UNIQUE,
+  `name` varchar(50) NOT NULL,
+  `password` varchar(50) NOT NULL,
+  `device` varchar(150) DEFAULT NULL,
+  `rollnumber` int(11) DEFAULT NULL,
+  `classname` varchar(15) NOT NULL,
+  `createdon` timestamp NOT NULL DEFAULT current_timestamp(),
+  `updatedon` timestamp NOT NULL DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+  Primary Key(id)
+);
+
+CREATE TABLE IF NOT EXISTS `Faculty` (
+  `id` int(10) NOT NULL AUTO_INCREMENT,
+  `username` varchar(20) NOT NULL UNIQUE,
+  `name` varchar(50) NOT NULL,
+  `password` varchar(32) NOT NULL,
+  `role` varchar(15) NOT NULL,
+  `createdon` timestamp NOT NULL DEFAULT current_timestamp(),
+  `updatedon` timestamp NOT NULL DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+  Primary Key(id)
+);
+
+CREATE TABLE IF NOT EXISTS `Subject` (
+  `id` int(10) NOT NULL AUTO_INCREMENT,
+  `name` varchar(100) DEFAULT NULL,
+  `code` varchar(15) NOT NULL,
+  `createdon` timestamp NOT NULL DEFAULT current_timestamp(),
+  `updatedon` timestamp NOT NULL DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+  Primary Key(id)
+);
+
+
+CREATE TABLE IF NOT EXISTS `AttendRecord` (
```

Comment:
Change all table names to lower snake case

Reply:
Created all tables to lower snake case